tests/rados/test_pg_dups_trimming.py

```diff
@@ -38,6 +38,7 @@ def run(ceph_cluster, **kw) -> int:
     pool_configs = config["pool_configs"]
     pool_configs_path = config["pool_configs_path"]
     test_image = config.get("container_image")
+    verify_inflation = config.get("verify_inflation", True)
```

Comment:
As verification of inflation of pg dups beyond the threshold is only possible with Pacific builds older than the fixed version, would it be better to keep the default value as `False`?

Reply:
Sounds good. Updated